html/js/desktop.js

```diff
@@ -532,31 +532,31 @@ function Desktop(elements) {
         removeBPMHardwareMapping: function(syncMode) {
           var instanceAndSymbol = "/pedalboard/:bpm"
           if (self.hardwareManager.removeHardwareMappping(instanceAndSymbol)) {
-              var source = syncMode === "link" ? "Ableton Link" : "MIDI"
-              new Notification('info', 'BPM addressing removed, incompatible with ' + source + ' sync mode', 8000)
-          }
-          self.pedalboardModified = true
+                var source = syncMode === "link" ? "Ableton Link" : "MIDI"
```

Comment:
Seems somene has different editor settings for spaces/tabs. Suggest you use `editorconfig`. 

Also, for some reason even though I fetch and merged your upstream changes this was left out. I triple checked it is in your master.